docs/Customizing Look and Functionality/Customizing the Keyboard/index.md

```diff
@@ -1,5 +1,11 @@
 # Customizing the Keyboard
-If you present keyboards in your app – such as in searchable interactions or another custom keyboard – you may wish to customize the keyboard for your users. The best way to do this is through the @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE]`ScreenManager`!@. For more information presenting keyboards, see the [Popup Menus and Keyboards guide](Displaying a User Interface/Popup Menus and Keyboards).
+@![iOS, android, javaEE, javaSE]
+If you present keyboards in your app – such as in searchable interactions or another custom keyboard – you may wish to customize the keyboard for your users. The best way to do this is through the !@@![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE]`ScreenManager`!@@![iOS, android, javaEE, javaSE]. For more information presenting keyboards, see the [Popup Menus and Keyboards guide](Displaying a User Interface/Popup Menus and Keyboards).
+!@
+
+@![javascript]
+The `ChoiceSetManager` used to customize the keyboard is currently missing from the SDL JavaScript Suite. This will be addressed in a future release. However you can still customize the keyboard through the `setGlobalProperties` RPC.
```

Comment:
```suggestion
The `ChoiceSetManager` used to customize the keyboard is currently missing from the SDL JavaScript Suite. This will be addressed in a future release. However you can still customize the keyboard through the `SetGlobalProperties` RPC.
```